<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>781</width>
      <height>641</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeView" name="treeView">
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Rename</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>9</y>
           <width>751</width>
           <height>241</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBox_3">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>remove n chars</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QLineEdit" name="lineEdit_3">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Suffix Text</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QCheckBox" name="checkBox_4">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>remove n chars</string>
            </property>
           </widget>
          </item>
          <item row="2" column="5">
           <widget class="QSpinBox" name="spinBox_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Prefix Text</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string>Append</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QSpinBox" name="spinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>Prepend</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
